packages/api-docs-builder/types/utils.types.ts

```diff
@@ -7,7 +7,7 @@ export interface Slot {
   default?: string;
 }
 
-export interface CssVariable {
+export interface ApiItemDescription {
```

Comment:
Made the name more generic to be reused with different types of API descriptions.